warehouse/query-core/src/main/java/datawave/query/lucene/visitors/AmbiguousUnfieldedTermsVisitor.java

```diff
@@ -0,0 +1,273 @@
+package datawave.query.lucene.visitors;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+import org.apache.lucene.queryparser.flexible.core.nodes.AndQueryNode;
+import org.apache.lucene.queryparser.flexible.core.nodes.FieldQueryNode;
+import org.apache.lucene.queryparser.flexible.core.nodes.GroupQueryNode;
+import org.apache.lucene.queryparser.flexible.core.nodes.OrQueryNode;
+import org.apache.lucene.queryparser.flexible.core.nodes.QueryNode;
+
+public class AmbiguousUnfieldedTermsVisitor extends BaseVisitor {
```

Comment:
Add a class comment as to what this visitor is looking for.

Reply:
Flxed.